docs/csharp/asynchronous-programming/index.md

```diff
@@ -1,73 +1,67 @@
+If you want the computer to execute instructions asynchronously, you must write asynchronous code. When you write client programs, you want the UI to be responsive to user input. Your application shouldn't freeze all interaction while downloading data from the web. When you write server programs, you don't want to block threads that might be serving other requests. Using synchronous code when asynchronous alternatives exist, hurts your ability to scale out less expensively. You pay for blocked threads.
 
-Successful modern applications require asynchronous code. Without language support, writing asynchronous code required callbacks, completion events, or other means that obscured the original intent of the code. The advantage of the synchronous code is that its step-by-step actions make it easy to scan and understand. Traditional asynchronous models forced you to focus on the asynchronous nature of the code, not on the fundamental actions of the code.
+Successful modern apps require asynchronous code. Without language support, writing asynchronous code requires callbacks, completion events, or other means that obscure the original intent of the code. The advantage of synchronous code is the step-by-step action that makes it easy to scan and understand. Traditional asynchronous models force you to focus on the asynchronous nature of the code, not on the fundamental actions of the code.
 
 ## Don't block, await instead
 
-The preceding code demonstrates a bad practice: constructing synchronous code to perform asynchronous operations. As written, this code blocks the thread executing it from doing any other work. It won't be interrupted while any of the tasks are in progress. It would be as though you stared at the toaster after putting the bread in. You'd ignore anyone talking to you until the toast popped.
+The previous code highlights an unfortunate programming practice: Writing synchronous code to perform asynchronous operations. The code blocks the current thread from doing any other work. The code doesn't interrupt the thread while there are running tasks. The outcome of this model is similar to you staring at the toaster after you put in the bread. In this scenario, you ignore any interruptions and don't start other tasks until the bread pops up. You might ignore a phone call, forget to take the butter and jam out of the fridge, or miss seeing a fire starting on the stove.
 
-Let's start by updating this code so that the thread doesn't block while tasks are running. The `await` keyword provides a non-blocking way to start a task, then continue execution when that task completes. A simple asynchronous version of the make a breakfast code would look like the following snippet:
+In your daily life, you don't want to overlook problems or miss opportunities. You want to both toast the bread and handle other concerns at the same time. The same is true with your code. 
```

Comment:
I'm not sure this is the best analogy - I think it's more about being "uninterruptible". But I'll leave it to Bill to decide.

Reply:
I removed the extra statement and adjusted the phrasing.